### PR TITLE
Transparency colour rules

### PR DESCRIPTION
**Pull request by stugoo**
Adding rules around text colour transparency

### CODE REVIEW

File: docs/_product/guidelines/colours-ui.md [hunk 1]
```diff
@@ -169,3 +169,11 @@ For maintaining colour contrast when using coloured text, some of the primary co
     <b>Note:</b> The purple primary colour is dark enough for use on most light backgrounds without the need for a darker variant.
   </div>
 </div>
+
+<div class="Note">
+  <div class="Note-description">
+    <b>Note:</b> The use of transparency in text colour should be avoided. If <em>absolutley</em> nesscary, use only white, or black at a opacity no lower than 80% to preserve readability and accessibility standards. <br>
```
Comment (summers1981FA) [inline docs/_product/guidelines/colours-ui.md]: Typo here @stugoo (nesscary)

Comment (paddyduke) [inline docs/_product/guidelines/colours-ui.md]: Would it be worth adding a note to manually check the contrast level of the resultant text colours against all the backgrounds they sit on?

Comment (summers1981FA) [reply docs/_product/guidelines/colours-ui.md]: Hey @stugoo, I still get `nesscary` on the latest change

Comment (stugoo) [reply docs/_product/guidelines/colours-ui.md]: Try it now @summers1981FA

**Pull request merged**